packages/agent/src/app.ts

```diff
@@ -627,17 +676,36 @@ export class App {
       this.log.info(`Forcing upgrade from ${MEDPLUM_VERSION} to ${targetVersion}`);
     }
 
+    if (upgradeInProgress && message.force) {
+      // If running, just cleanup the file since it could be that the cleanup failed for whatever reason
+      if (isAppRunning('medplum-upgrading-agent')) {
+        removePidFile(getPidFilePath('medplum-upgrading-agent'));
```

Comment:
Need to change this and check all references to `removePidFile` since we simplified the signature